features/login_github_redirect.feature

```diff
@@ -0,0 +1,151 @@
+Feature: after a user logins with their github id they are redirected to the page they wanted to view prior to login
+
+  As a user who first opens the app
+  So that I can efficiently login and browse the app
+  I want to return to the page I requested to login
+
+Background: users, orgs and apps have been added to database
+  Given the following apps exist:
+        | name  | description | org_id | status  |
+        | app1  | test        | 1      | pending |
+        | app2  | test        | 1      | pending |
+        | app3  | test        | 1      | pending |
+
+    And the following orgs exist:
+        | name | contact_id |
+        | org1 | 1          |
+        | org2 | 1          |
+        | org3 | 1          |
+
+    And the following users exist:
+        | name  | github_uid      | email         |
+        | user1 | esaas_developer | test@user.com |
+        | user2 |                 | test@user.com |
+        | user3 |                 | test@user.com |
+
+
+  And I am not logged in
+
+Scenario: not logged in so should see login with Github from users page
+  #Story ID: #152298585
+  Given I am on the users page
+  Then I should see "Log in with GitHub"
+ 
+Scenario: not logged in so should see login with Github from orgs page 
+  #Story ID: #152298585
+  Given I am on the orgs page
+  Then I should see "Log in with GitHub"
+  
+Scenario: not logged in so should see login with Github from create page 
+  #Story ID: #152298585
+  Given I am on the create page
+  Then I should be on the login page
+  # Then I should see "Log in with GitHub"
+
+Scenario: login with Github from orgs page and redirect to orgs
+  #Story ID: #152298585
+  Given I am not logged in
```

Comment:
I see a lot of "Given I am not logged in" and "And I am on the apps page" at the beginning of each scenario. I think it is a better idea put those into the background.